db/migrations/20241104115955_add_cancel_attempted_to_tasks.down.sql

```diff
@@ -1,5 +1,19 @@
 BEGIN;
 
+DROP VIEW if exists ready_tasks;
+
 ALTER TABLE tasks DROP COLUMN IF EXISTS cancel_attempted;
 
+CREATE OR REPLACE VIEW ready_tasks AS
+SELECT *
+FROM tasks
+WHERE started_at IS NULL
+  AND (status != 'canceled' OR status is null)
+  AND id NOT IN (
+    SELECT task_id
+    FROM task_dependencies JOIN tasks ON dependency_id = id
+    WHERE finished_at IS NULL
+)
+ORDER BY priority DESC, queued_at ASC;
+
 COMMIT;
```

Comment:
This was required as the cancel_attempted column is part of the ready_tasks view.  It hadn't been hit before because teh ready_tasks view wasn't ever recreated  after this column was added.  

This means the view didn't actually include this column until this migration was added. 

I wonder if we should explicitly list the columns on the ready_task view to avoid this situation.  Thoughts?

Reply:
So the ready_tasks view creation would become:

```
CREATE OR REPLACE VIEW ready_tasks AS
SELECT id, org_id, token,type, payload, status, error, queued_at, started_at, finished_at, request_id, account_id, retries, next_retry_time, priority, object_uuid, object_type
FROM tasks t
```

we could probably even reduce the amount of columns in the list i would think

Reply:
sounds good to me!

Reply:
this will require two migrations, one switching to this query, and then once this is in production, one dropping the ready_tasks view.  I'll go ahead and open the 2nd PR and we'll just hold it until this one is promoted to prod